extensions/machine-learning/src/views/models/prediction/columnsTable.ts

```diff
@@ -366,11 +367,11 @@ export class ColumnsTable extends ModelViewBase implements IDataComponent<Predic
 		return [];
 	}
 
-	private createWarningButton(message: string): azdata.ButtonComponent {
+	private createWarningButton(): azdata.ButtonComponent {
 		const warningButton = this._modelBuilder.button().withProperties({
 			width: '16px',
 			height: '16px',
-			title: message,
+			title: constants.columnDataTypeMismatchWarningHelper,
```

Comment:
the message was supposed to be different for input and output.

Reply:
It IS different. **columnDataTypeMismatchWarningHelper** is NOT the dialog title, it's the title attribute on each button which tells the user to click for more info. Then in each dialog, depending on output or input row, the message is as follows:
"The data type of the source table column does not match the required [ input | output ] field’s type."